python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -646,6 +671,11 @@ def combine(self, cache, struct, outputId):
         calib = self.combination.run(dataRefList, expScales=struct.scales.expScales,
                                      finalScale=struct.scales.ccdScale)
 
+        if not hasattr(calib, "getMetadata"):
+            calib = afwImage.DecoratedImageF(calib.getImage())
```

Comment:
Is there a `makeDecoratedImage` that could be used instead of being explicit about the template?